sigchanyzer.go

```diff
@@ -48,17 +48,25 @@ func run(pass *analysis.Pass) (interface{}, error) {
 			return
 		}
 		var chanDecl *ast.CallExpr
+
+		// track whether call.Args[0] is a *ast.CallExpr. This is true when make(chan os.Signal) is passed directly to signal.Notify
+		wasCallExpr := false
```

Comment:
Let's make this name more direct and more legible. Let's rename this to `inlinedChanMake := false`